src/react-components/src/components/Image/Image.tsx

```diff
@@ -34,5 +38,23 @@ export default function Image(props: ImageProps) {
         alt: isDecorative ? "" : altText,
     };
 
-    return <img {...imageProps} />;
+    return (
+        <>
+            <img {...imageProps} />
+            {(imageCaption || imageCredit) && (
+                <div className={bem(image_base_class, [], "annotation")}>
+                    {imageCaption && (
+                        <p className={bem(image_base_class, [], "caption")}>
```

Comment:
Do these paragraphs need `bem()` classes even if they're being selected based on their position in the annotation div?

Reply:
Giving all three (wrapper, child, second child) their own BEM classes allows at the integration level for a project to decide to style them differently if they so choose with a clear hook to do so. I think what you've done here makes sense.

Reply:
Thanks for providing that context is helpful! I will definitely err towards including BEM classes.